src/site/content/en/blog/build-for-webusb/index.md

```diff
@@ -0,0 +1,1018 @@
+also been backported to USB 2.0 devices as part of version 2.1. Declaring
+support for WebUSB starts with including the following Platform Capability
+Descriptor in the BOS descriptor:
+
+<div class="w-table-wrapper">
+  <table>
+    <tr>
+    <th style="width: 33%">Value</th>
+    <th style="width: 33%">Field</th>
+    <th style="width: 33%">Description</th>
+    </tr>
+    <tr>
+    <td colspan="3">Binary device Object Store descriptor</td>
+    </tr>
+    <tr>
+    <td><code>0x05</code></td>
+    <td>bLength</td>
+    <td>Size of this descriptor</td>
+    </tr>
+    <tr>
+    <td><code>0x0F</code></td>
+    <td>bDescriptorType</td>
+    <td>Binary device Object Store descriptor</td>
+    </tr>
+    <tr>
+    <td><code>0x001D</code></td>
+    <td>wTotalLength</td>
+    <td>Total length of this series of descriptors</td>
+    </tr>
+    <tr>
+    <td><code>0x01</code></td>
+    <td>bNumDeviceCaps</td>
+    <td>Number of device capability descriptors in the BOS</td>
+    </tr>
+    <tr>
+    <td colspan="3">WebUSB platform capability descriptor</td>
+    </tr>
+    <tr>
+    <td><code>0x18</code></td>
+    <td>bLength</td>
+    <td>Size of this descriptor</td>
+    </tr>
+    <tr>
+    <td><code>0x10</code></td>
+    <td>bDescriptorType</td>
+    <td>Device capability descriptor</td>
+    </tr>
+    <tr>
+    <td><code>0x05</code></td>
+    <td>bDevCapabilityType</td>
+    <td>Platform capability descriptor</td>
+    </tr>
+    <tr>
+    <td><code>0x00</code></td>
+    <td>bReserved</td>
+    <td></td>
+    </tr>
+    <tr>
+    <td><code>{0x38, 0xB6, 0x08, 0x34, 0xA9, 0x09, 0xA0, 0x47, 0x8B, 0xFD, 0xA0, 0x76, 0x88, 0x15, 0xB6, 0x65}</code></td>
+    <td>PlatformCapablityUUID</td>
+    <td>WebUSB platform capability descriptor GUID in little-endian format</td>
+    </tr>
+    <tr>
+    <td><code>0x0100</code></td>
+    <td>bcdVersion</td>
+    <td>WebUSB descriptor version 1.0</td>
+    </tr>
+    <tr>
+    <td><code>0x01</code></td>
+    <td>bVendorCode</td>
+    <td>bRequest value for WebUSB</td>
+    </tr>
+    <tr>
+    <td><code>0x01</code></td>
+    <td>iLandingPage</td>
+    <td>URL for landing page</td>
+    </tr>
+  </table>
+</div>
+
+{% Aside %}
+The UUID above would be written as `{3408b638-09a9-47a0-8bfd-a0768815b665}`
+however when sent as part of a USB descriptor its component fields must be sent
+in little-endian order. This transformation is complex so the proper encoding is
+given here for reference. See [RFC4122](https://tools.ietf.org/html/rfc4122).
+{% endAside %}
+
+The platform capability UUID identifies this as a [WebUSB Platform Capability
+descriptor], which provides basic information about the device. For the browser
+to fetch more information about the device it uses the `bVendorCode` value to
+issue additional requests to the device. The only request currently specified is
+`GET_URL` which returns a [URL descriptor].
+These are similar to string descriptors however are designed to encode URLs in
```

Comment:
```suggestion
These are similar to string descriptors but are designed to encode URLs in
```

Reply:
Is line 487 meant to continue the previous paragraph or start a new one?

Reply:
Done. It wasn't a separate paragraph in https://developers.google.com/web/fundamentals/native-hardware/build-for-webusb